dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/src/org/apache/cxf/transport/http/URLConnectionHTTPConduit.java

```diff
@@ -393,6 +409,7 @@ protected InputStream getInputStream() throws IOException {
                         in = connection.getInputStream();
                     } catch (IOException ex) {
                         // ignore
+        	        LOG.fine("Ignoring unexpected exception in getInputStream(): " + ex); // Liberty Change
```

Comment:
It seems to me that this code section might be incorrect (but would not cause a problem in most cases).  They are attempting to get the "Error Stream".  Shouldn't the if check here be `if (in != null)`?

Also, I'd suggest changing your log message to `"Ignoring unexpected exception in getInputStream() when an error stream is present: "`